docs/jwst/assign_wcs/reference_files.rst

```diff
@@ -29,53 +29,132 @@ reftype                                     description
 **ote**                Transform through the Optical Telescope Element               NIRSPEC
 **specwcs**            Wavelength calibration models                                 MIRI, NIRCAM, NIRISS
 **regions**            Stores location of the regions on the detector                MIRI
+**v2v3**               Transform from MIRI instrument focal plane to V2V3 plane      MIRI
```

Comment:
This reference file type was used in the initial implementation of the pipeline, before we had any input from the teams. It is not used any more.